staging/src/k8s.io/kube-openapi/README.md

```diff
@@ -0,0 +1,14 @@
+# Kube OpenAPI
+
+This repo is the home for Kubernetes OpenAPI discovery spec generation. The goal 
+is to support a subset of OpenAPI features to satisfy kubernetes use-case but 
+implement those subset with little to no assumption about the structure of the 
+code or routes. Thus, there should be no kubernetes specific code in this repo. 
+
+
+There are two main parts: 
+ - A model generator that goes through .go files, find types and generate model 
```

Comment:
Am no native speaker, so I might be wrong: A model generator that goes through .go files, finds and generates model definitions.